go.mod

```diff
@@ -147,3 +169,8 @@ replace (
 	github.com/gogo/protobuf v1.3.0 => github.com/gogo/protobuf v1.3.2
 	github.com/gogo/protobuf v1.3.1 => github.com/gogo/protobuf v1.3.2
 )
+
+// Required by ??? TODO
+replace k8s.io/cri-api => k8s.io/cri-api v0.25.8
+
+replace github.com/cilium/ebpf => github.com/cilium/ebpf v0.0.0-20230320181651-5ea5368f9823
```

Comment:
Please add a comment like:

```
// Required by traceloop, Will be in future release cilium/ebpf v0.10.1
// https://github.com/cilium/ebpf/commit/5ea5368f9823c13e71146a6c2cca84ad60a5e26a
```

cc @alban

Reply:
Added it in #1233